i18n/en/faq.md

```diff
@@ -114,7 +114,7 @@ In early/mid 2015, one of the developers with push privileges on GitHub pushed a
 
 These actions offended many in the I2P community, including the developers, and nearly ended the C++ project.
 
-In the fall of 2015, along came anonimal who, not wanting to see everyone's work to go to waste, revived the project through contributions of their own and by reigning-in development. An open invitation for all remaining active developers to meet and discuss i2pd's future was then given. i2pd's first author never showed but the act of meeting apparently rustled i2pd's feathers to the point where he [retaliated](https://github.com/PurpleI2P/i2pd/issues/279) and began to work on GitHub again - but this time within an ```openssl``` branch (which turned out to be the Bitbucket repository) instead of the community-driven ```master``` branch.
+In the fall of 2015, along came anonimal who, not wanting to see everyone's work go to waste, revived the project through contributions of their own and by reigning-in development. An open invitation for all remaining active developers to meet and discuss i2pd's future was then given. i2pd's first author never showed but the act of meeting apparently rustled i2pd's feathers to the point where he [retaliated](https://github.com/PurpleI2P/i2pd/issues/279) and began to work on GitHub again - but this time within an ```openssl``` branch (which turned out to be the Bitbucket repository) instead of the community-driven ```master``` branch.
```

Comment:
Please try not to mix English changes with French changes (at least not within the same commit).

Reply:
yes, I will change that

Reply:
Sorry @anonimal , I had not seen monero-project/kovri-site#18 , but why did you mention it?

Reply:
Hi @woodyjon. I mention it because it may be abandoned. @Dridou are you still around?